Vagrantfile

```diff
@@ -16,7 +16,7 @@ ARCH="$(uname -m | sed 's|i686|386|' | sed 's|x86_64|amd64|')"
 export DEBIAN_PRIORITY=critical
 export DEBIAN_FRONTEND=noninteractive
 export DEBCONF_NONINTERACTIVE_SEEN=true
-APT_OPTS="--yes --force-yes --no-install-suggests --no-install-recommends"
+APT_OPTS="--yes --force-yes --no-install-suggests --no-install-recommends -o Dpkg::Options::=\"--force-confdef\" -o Dpkg::Options::=\"--force-confold\""
```

Comment:
I actually liked changing `--yes --force-yes` to `--assume-yes` as it's more graceful, and the man page actually suggests not using force.